internal/model.go

```diff
@@ -273,6 +273,24 @@ func (i *InstanceDetails) GetCompassRuntimeId() string {
 	return *i.CompassRuntimeId
 }
 
+func (i *InstanceDetails) GetRuntimeResourceName() string {
+	name := i.RuntimeResourceName
+	if name == "" {
+		// fallback to runtime ID
+		name = i.RuntimeID
+	}
+	return name
+}
+
+func (i *InstanceDetails) GetRuntimeResourceNamespace() string {
+	namespace := i.KymaResourceNamespace
```

Comment:
naming check, we already have Kyma CRs and soon there will be Runtime CRs. Is `KymaResourceNamespace` a correct field to get the namespace of Runtime CR?

Reply:
The question is if there is possibility, that Kyma CR would use different namespace than Runtime CR.
If so, we should add RuntimeResourceNamespace.